javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,17 @@
+const fullName = 'Cathy Garcia';
+const isCool = true;
+const totalPets = 1;
+const unknown = null;
+let never;
+
+console.log('Value of fullName:', fullName);
+console.log('Value of isCool', isCool);
+console.log('Value of totalPets', totalPets);
+console.log('Value of unknown:', unknown);
+console.log('Value of never', never);
+console.log('type of:', typeof fullName);
+console.log('type of:', typeof isCool);
+console.log('type of:', typeof totalPets);
+console.log('type of:', typeof unknown);
+console.log('type of', typeof never);
+console.log('test');
```

Comment:
Delete this console log. I added this when I was testing your commit, but it's not actually apart of the exercise.